readme.md

```diff
@@ -48,10 +48,18 @@ Run the tests:
 REPO_DIR=<path to your integration tests repo> $REPO_DIR/bats/run_tests.sh
 ```
 
-Optional variables for `run_tests.sh` script:
+To run tests on your branch you should define variables:
 - `GIT_REPO=<git repository URL>`, default value: `https://github.com/pyrsia/pyrsia.git`
 - `GIT_BRANCH=<branch repository name>`, default value: `main`
 
+For example, to run tests from the folder with cloned `pyrsia-integration-tests` repo, you can use following commands:
+```sh
+export GIT_REPO=<your_forked_pyrsia_repo_url>
```

Comment:
We don't need to use `export`, but just can pass variables when running scripts like L.122 in the same file. Isn't it easier to set and change every time?
For example, here is a command I usually use.
```sh
REPO_DIR=/path/to/pyrsia-integration-tests GIT_REPO="https://github.com/ihcomega56/pyrsia.git" GIT_BRANCH="apply-useful-messages-to-all-errors" TEST_LOG_LEVEL=debug ./bats/run_tests.sh
```

Reply:
Maybe it would be enough if you make change like this.

- `GIT_REPO=<your_forked_pyrsia_repo_url>`, default value: `https://github.com/pyrsia/pyrsia.git`
- `GIT_BRANCH=<branch_name_to_test>`, default value: `main`

Reply:
An example is the best explanation. 
We could just make our example simpler, as you suggested.